src/internet/radiobrowser/radiobrowserservice.cpp

```diff
@@ -433,7 +433,7 @@ void RadioBrowserService::ResolveStationUrl(const QUrl& original_url) {
     Song ret;
     ret.set_valid(true);
     ret.set_title(item["name"].toString());
-    QUrl url(item["url"].toString());
+    QUrl url(item["url_resolved"].toString());
```

Comment:
Is `url_resolved` always populated? I wonder if we want to fall back on `url` if it's empty.

Reply:
The API documentation itself doesn't mark any field as optional: https://de1.api.radio-browser.info/#Struct_station
I had a look at the API source itself and the url_resolved field comes from a database field called "UrlCache" that is used internally for other things, eg. to determine if a station actually works and looking for duplicate stations.
Still, I'm adding the fall back just to stay on the safe side.